packages/gatsby-source-strapi/README.md

```diff
@@ -77,6 +77,7 @@ const strapiConfig = {
   accessToken: process.env.STRAPI_TOKEN,
   collectionTypes: ["article", "company", "author"],
   singleTypes: [],
+  maxParallelRequests: 5, // (Optional) Default: Number.POSITIVE_INFINITY
```

Comment:
Is this a reasonable default? Is it worth doing some testing to find something more finite? E.g. performs well but won't knock over strapi?

Reply:
Left the `Number.POSITIVE_INFINITY` as a default value to effectively have no limit, as it is currently working like this, did not want to change the behaviour for users that will upgrade and did not have this need.

You maintain in for long time, so I guess you may have a better view of it, this is easily changed.

To give more context also, for us specifically the no-limit was causing:
1. sometimes Strapi server to go down and restart, but for this to happen I guess you need a CMS with heavy enough content. A few Strapi pages will not have any problem
2. suffocating (home) router connections with many parallel requests, resulting in the router dropping connections

Reply:
Yeah...I agree leaving it is most the same. Let's just leave it and this can always be revisited.